### PR TITLE
New package: font-inter-3.19

### PR DESCRIPTION
**Pull request by linkert**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES?**

If one would like to reduce some of the absolute pinnacle of screen specific typeface into "aesthetics", well that's under appreciating something like Inter. An absolute beast of a typeface within the space it exists. Anyone should be able to enjoy the detail and work that's gone into this typeface. Check out the website, it provides far more justice and detail as to why it exists, it's goals and strengths than I'm able to provide here.

I'd like it to be merged because:

- The insane quality this typeface offers. I'm gonna use it for GTK applications, for PDF exports through pandoc texlive stuff and whatnot.
- It's actively developed. Some of us do care about typography and want to receive updates in a modern, convenient manner just as with any other software on our systems. Like Iosevka which is in the repos — love seeing it in my updates, always checkout the changes, fixes and features it brings.
- It might add some new glyphs? — haven't done a diff. Maybe, for instance the insert key symbol, that's one I rarely see [⎀](https://rsms.me/inter/glyphs/?g=insertionsymbol) along with other modifier keys in a cohesive way
- I have been a Void Linux user since 2019  — want to start nice and easy with the role as a package maintainer :)

### CODE REVIEW

**Comment by classabbyamp** [comment]
you can just download the font to `/usr/local/share/fonts` or `~/.local/share/fonts` if you think it's so amazing...

**Comment by abenson** [comment]
Please [squash the commits](https://www.git-tower.com/learn/git/faq/git-squash) into a single commit.

**Comment by ahesford** [comment]
This seems like a pretty thorough font; seems OK to me to package it.

**Comment by linkert** [comment]
> you can just download the font to `/usr/local/share/fonts` or `~/.local/share/fonts` if you think it's so amazing...

I know but then again I addressed that point.

> ..receive updates in a modern, convenient manner

**Comment by linkert** [comment]
> Please [squash the commits](https://www.git-tower.com/learn/git/faq/git-squash) into a single commit.

OK, times up for this morning. I have now faffed about with git frantically trying things out. Hopefully nothing is too messed up now ;) 

I went `git rebase -i HEAD~2` since I figured it's the last two commits of the branch — this gave me the option to rebase/squash these two commits. (below is from git log)
```
* 461dfa7798 2023-03-18 22:15:19 +0100 New package: font-inter-3.19 (Robin)
* b275f0f143 2023-03-18 22:01:09 +0700 font-sil-padauk: update to 5.001. (upstream/master, origin/master, origin/HEAD, master) (Đoàn Trần Công Danh)
```
Nope, not what I was looking to do.


Looking at my git log after my frantic behavior is a bit of a mess.
```
*-.   e1d8d0f37d 2023-03-19 08:42:19 +0100 WIP on font-inter: 1033692c9b New package: font-inter-3.19 (refs/stash) (Robin)
|\ \  
| | * f5eb6fff54 2023-03-19 08:42:19 +0100 untracked files on font-inter: 1033692c9b New package: font-inter-3.19 (Robin)
| * 10561181bb 2023-03-19 08:42:19 +0100 index on font-inter: 1033692c9b New package: font-inter-3.19 (Robin)
|/  
* 1033692c9b 2023-03-19 08:32:16 +0100 New package: font-inter-3.19 (HEAD -> font-inter) (Robin)
| * 9bb05ad54f 2023-03-18 23:20:02 +0100 font-inter: Added vlicense (origin/font-inter) (Robin)
| * 461dfa7798 2023-03-18 22:15:19 +0100 New package: font-inter-3.19 (Robin)
|/  
* b275f0f143 2023-03-18 22:01:09 +0700 font-sil-padauk: update to 5.001. (upstream/master, origin/master, origin/HEAD, master) (Đoàn Trần Công Danh)
...
```

Does not seem that 9bb05ad54f is part of HEAD then? Is that it?

I'll look into it tonight otherwise, thank you for your patience!

**Comment by paper42** [comment]
git rebase upstream/master assuming upstream is your void-linux/void-packages remote

**Comment by linkert** [comment]
> git rebase upstream/master assuming upstream is your void-linux/void-packages remote

Thanks, gonna go over to my neighbor tonight for some guidance. This morning I thought a combination of the guide for squashing and simply reading the feedback given by git would be enough to get by.

**Comment by linkert** [comment]
> Please [squash the commits](https://www.git-tower.com/learn/git/faq/git-squash) into a single commit.

Lovely neighbor of mine explained where I got lost and now I do believe all is in order?

File: srcpkgs/font-inter/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'font-inter'
+pkgname=font-inter
+version=3.19
+revision=1
+create_wrksrc=yes
```
Comment (sgn) [inline srcpkgs/font-inter/template:5]: Remove this line
```suggestion
```

File: srcpkgs/font-inter/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+distfiles="https://github.com/rsms/inter/releases/download/v${version}/Inter-${version}.zip"
+checksum=150ab6230d1762a57bebf35dfc04d606ff91598a31d785f7f100356ecdcc0032
+
+font_dirs="/usr/share/fonts/OTF"
+
+do_install() {
+	vmkdir usr/share/fonts/OTF
+	vcopy *Desktop/*.otf usr/share/fonts/OTF
+	vlicense LICENSE.txt LICENSE.md
+}
```
Comment (sgn) [inline srcpkgs/font-inter/template:20]: _I_ (personally) think it's better to not put too much fonts into `/usr/share/fonts/OTF`, but split them into sub-directory by category, publisher, etc...

File: srcpkgs/font-inter/template (hunk 1 above)
Comment (linkert) [reply srcpkgs/font-inter/template:5]: Will do

File: srcpkgs/font-inter/template (hunk 2 above)
Comment (linkert) [reply srcpkgs/font-inter/template:20]: Not at all against this suggestion, but I don't think I have found any other font package considering this? Therefore I decided to simply follow along. Do you know of any?

If we are going to organize by sub-categories there should probably be some syntax, some pattern right?

Overkill?:
```
../OTF/variable/sans/rsms/inter/*
../OTF/static/sans/{publisher}/foo/*
../OTF/static/monospace/{publisher}/bar/*
```

Comment (classabbyamp) [reply srcpkgs/font-inter/template:20]: I would just put them in /usr/share/fonts/inter

**Comment by linkert** [comment]
I got two commits going again.. I know, and this is not a git support forum so I'm reluctant to ask for help on here. Basically I went a bit fast with the changes, committed and did a force push (had some thought that a force push squaches all into one commit, this is wrong). I now when I try do an interactive rebase on master but get the following `error: cannot 'squash' without a previous commit` — thanks git, real helpful.

Gonna pull the neighbor-card once more - probably Wednesday.

**Comment by linkert** [comment]
All should be in order - does anyone get pinged when a commit is pushed?

**Pull request merged**